apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -268,18 +245,17 @@ export class SpecialAnnouncementActionBlock extends Component {
       <TwoColumnActionBlock
         id="teacher-nomination-announcement"
```

Comment:
This block is no longer nomination announcement, maybe replace the id with something like `id="teacher-application-announcement` (the block above it is `teacher-application-continue-announcement`).